cmd/deploy/deploy.go

```diff
@@ -287,6 +314,19 @@ func (dc *deployCommand) runDeploy(ctx context.Context, cwd string, opts *Option
 	return nil
 }
 
+func setManifestEnvVars(service, registry, repository, image, tag string) {
+	os.Setenv(fmt.Sprintf("build.%s.registry", service), registry)
+	os.Setenv(fmt.Sprintf("build.%s.repository", service), repository)
+	os.Setenv(fmt.Sprintf("build.%s.image", service), image)
+	os.Setenv(fmt.Sprintf("build.%s.tag", service), tag)
+
+	log.Debug("manifest env vars set")
+}
+
+func expandManifestEnvVars(manifest string) string {
```

Comment:
Minor thing: Can we change the param name? If I'm not wrong, this is not intended to receive an Okteto manifest to expand any env variable on it, right? I think it can be misleading. As it is a one-liner function maybe we can just replace its use. Anyways, no strong opinion, you can leave it as it is :)